<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>deviceConsole</class>
 <widget class="QDialog" name="deviceConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="verticalSpacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnRst">
       <property name="text">
        <string>+360</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/res/image/icon/332.png</normaloff>:/res/image/icon/332.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCls">
       <property name="text">
        <string>-360</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/res/image/icon/332.png</normaloff>:/res/image/icon/332.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnWrite">
       <property name="text">
        <string>Write</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/res/image/icon/218.png</normaloff>:/res/image/icon/218.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRead">
       <property name="text">
        <string>Read</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/res/image/icon/219.png</normaloff>:/res/image/icon/219.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClr">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="icon">
        <iconset resource="../res/res.qrc">
         <normaloff>:/res/image/icon/empty.png</normaloff>:/res/image/icon/empty.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>*IDN?</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnClr</sender>
   <signal>clicked()</signal>
   <receiver>listWidget</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>413</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
